### ISSUE TITLE
hi

### ISSUE THREAD

**Issue by hihihihihiray** [tealyy/ss_diontimmer-shiawase#1]
your map doesn’t work did you test it ingame

**Comment by tealyy** [comment]
yeah i know it dont work. u might be the more notes gamepass for it to work. for now im just gonna close this repository
